tf_encrypted/keras/models/sequential.py

```diff
@@ -101,3 +104,113 @@ def layers(self):
     if layers and isinstance(layers[0], InputLayer):
       return layers[1:]
     return layers[:]
+
+  def set_weights(self, weights, sess=None):
+    """ Sets the weights of the model.
+    Arguments:
+      weights: A list of Numpy arrays with shapes and types
+          matching the output of model.get_weights()
+      sess: tfe session"""
+
+    # Updated weights for each layer
+    for layer in self.layers:
+      num_param = len(layer.weights)
+      layer_weights = weights[:num_param]
+      # Define keras weights as private placeholder
+      tfe_weights_pl = [tfe.define_private_placeholder(w.shape)
```

Comment:
I'm pretty sure everything from here through line 128 can be replaced by `layer.set_weights(layer_weights, sess)`

Reply:
ah correct, I am sure we can also do this :)